strawberry/fastapi/router.py

```diff
@@ -38,36 +45,34 @@ async def __get_root_value():
 
     @staticmethod
     def __get_context_getter(
-        custom_getter: Callable[..., Optional[Union[BaseContext, Dict[str, Any]]]]
-    ) -> Callable[..., Union[BaseContext, Dict[str, Any]]]:
+        custom_getter: Callable[..., Optional[CustomContext]]
+    ) -> Callable[..., CustomContext]:
         def dependency(
-            custom_getter: Optional[Union[BaseContext, Dict[str, Any]]],
+            custom_getter: Optional[CustomContext],
             background_tasks: BackgroundTasks,
             request: Request = None,
             response: Response = None,
             ws: WebSocket = None,
-        ) -> Union[
-            BaseContext, Dict[str, Union[Any, BackgroundTasks, Request, WebSocket]]
-        ]:
+        ) -> MergedContext:
+            default_dict = {
```

Comment:
Perhaps a better var name here would be `default_context`? I was initially confused thinking this was `defaultdict` from the standard library.